crypto/BRCryptoWalletManagerClient.c

```diff
@@ -851,41 +874,41 @@ cwmWalletManagerEventAsETH (BREthereumClientContext context,
             break;
         }
 
-        case EWM_EVENT_SYNC_STARTED:
-            cwmEvent = (BRCryptoWalletManagerEvent) {
-                CRYPTO_WALLET_MANAGER_EVENT_SYNC_STARTED
-            };
-            cwm->listener.walletManagerEventCallback (cwm->listener.context,
-                                                      cryptoWalletManagerTake (cwm),
-                                                      cwmEvent);
+        case EWM_EVENT_CHANGED:
```

Comment:
Inferring this seems tricky. Interested in knowing what the issue was with the STARTED/STOPPED events.

Reply:
(Actually, I was inclined to take it further - to sync started/stopped.)  The event is a change of state - the action is to do what you please based on a) the state you are exiting and b) the state you are entering.